pyiso/isone.py

```diff
@@ -188,3 +190,68 @@ def parse_json_load_data(self, data):
                 return data['FiveMinSystemLoads']['FiveMinSystemLoad']
         except (KeyError, TypeError):
             raise ValueError('Could not parse ISONE load data %s' % data)
+
+    def get_lmp(self, zone_name, latest=True, start_at=False, end_at=False, **kwargs):
+
+        # TODO, handle kwargs including latest, etc
+        loc_dict = [a for a in self.locations if a['LocationName'] == '.Z.%s' % zone_name][0]
+        locationid = loc_dict['LocationID']
+        if latest:
+            endpoint = '/fiveminutelmp/current/location/%s.json' % locationid
+            price = self.fetch_data(endpoint, self.auth)['FiveMinLmp'][0]['LmpTotal']
+            return price
+        else:
+            day = start_at.strftime('%Y%m%d')
+            if not end_at:
+                end_at = start_at + timedelta(days=1)
+
+            endpoint = '/fiveminutelmp/day/%s/location/%s.json' % (day, locationid)
+            response = self.fetch_data(endpoint, self.auth)
+
+            lmp_dict = {}
+            for item in response['FiveMinLmps']['FiveMinLmp']:
+                # '%Y-%m-%dT%H:%M:%S.%f%z'
+                current_time = parser.parse(item['BeginDate']).astimezone(pytz.utc)
+                if start_at <= current_time <= end_at:
+                    lmp_dict[current_time] = item['LmpTotal']
+            return lmp_dict
+
+    locations = [
```

Comment:
This is pulled from the ISONE api, endpoint 'locations'.  Since it doesn't change very often, particularly for the zones (MAINE, NEWHAMPSHIRE, etc)  I thought it would be better to store locally rather than hit the API for it each time.